build.gradle

```diff
@@ -1,93 +1,113 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java Library project to get you started.
- * For more details take a look at the Java Libraries chapter in the Gradle
- * User Manual available at https://docs.gradle.org/6.2.2/userguide/java_library_plugin.html
- */
-
 plugins {
-    // Apply the java-library plugin to add support for Java Library
-    id 'java-library'
-    id "com.github.johnrengelman.shadow" version "5.2.0"
-    id("maven-publish")
-    id "com.jfrog.bintray" version "1.8.5"
+    id "java-library"
+    id "maven-publish"
+    id "signing"
 }
 
-def pluginVersion = '2.1.5'
+group = "io.snyk.code.sdk"
+archivesBaseName = "snyk-code-client"
+version = "2.1.5-SNAPSHOT"
+ext.isReleaseVersion = !version.endsWith("SNAPSHOT")
 
-publishing {
-/*
-    repositories {
-        maven {
-            name = "JavaClient"
-            url = uri("https://maven.pkg.github.com/artsiomch/java-client-dev")
-            credentials {
-                username = System.getenv("githubUsername")
-                password = System.getenv("githubToken")
-            }
-        }
-    }
-*/
-    publications {
-        maven(MavenPublication) {
-            //artifact(shadowJar)
-            groupId = 'ai.deepcode'
-            artifactId = 'java-client'
-            version = pluginVersion
-            from(components.java)
-        }
-    }
+repositories {
+    mavenCentral()
 }
 
-bintray {
-    user = System.getenv('bintrayUser')
-    key = System.getenv('bintrayKey')
-    publications = ['maven']
-
-    pkg {
-        repo = 'maven-releases'
-        name = 'java-client'
-        userOrg = 'artsiomch'
-        version {
-            name = pluginVersion
-            released  = new Date()
-        }
-    }
-}
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+
 
 java {
+    withJavadocJar()
```

Comment:
do we need that?

Reply:
yes, Maven requires it for IDE lookup
https://maven.apache.org/repository/guide-central-repository-upload.html#requirements